presentation/src/main/java/me/androidbox/presentation/login/screen/LoginScreen.kt

```diff
@@ -0,0 +1,150 @@
+package me.androidbox.presentation.login.screen
+
+import android.util.Log
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.foundation.text.ClickableText
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.*
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import androidx.hilt.navigation.compose.hiltViewModel
+import me.androidbox.component.R
+import me.androidbox.component.general.PasswordEntry
+import me.androidbox.component.general.TaskButton
+import me.androidbox.component.general.UserInputEntry
+import me.androidbox.component.ui.theme.BusbyTaskyTheme
+import me.androidbox.component.ui.theme.backgroundInputEntry
+import me.androidbox.presentation.login.viewmodel.LoginViewModel
+
+@Composable
+fun LoginScreen(
+    modifier: Modifier = Modifier,
+    loginViewModel: LoginViewModel = hiltViewModel()
+) {
+
+    Column(modifier = modifier
+        .fillMaxSize()
+        .background(Color.Black)) {
+
+        Spacer(modifier = modifier.height(48.dp))
+
+        Text(
+            modifier = modifier.fillMaxWidth(),
+            text = "Welcome Back!", color = Color.White, fontSize = 28.sp, fontWeight = FontWeight.Bold,
+            textAlign = TextAlign.Center)
+
+        Spacer(modifier = modifier.height(42.dp))
+
+        Box {
+            Column(
+                modifier = modifier
+                    .fillMaxSize()
+                    .background(
+                        color = Color.White,
+                        shape = RoundedCornerShape(topStart = 32.dp, topEnd = 32.dp)
+                    )
+                    .padding(horizontal = 16.dp)
+            ) {
+                Spacer(modifier = modifier.height(50.dp))
+                UserInputEntry(
+                    modifier = Modifier
+                        .fillMaxWidth()
+                        .background(
+                            shape = RoundedCornerShape(10.dp),
+                            color = MaterialTheme.colorScheme.backgroundInputEntry
+                        ),
```

Comment:
@philipplackner @MKiperszmid 

So for modifiers, we should pass all modified properties i.e. `height`, `background`, `fullmaxWidth`, etc to the composable?

Reply:
Exactly

I usually don't receive a Modifier on Screens. Since they are usually fillMaxSize() and such